nav2_controller/include/nav2_controller/nav2_controller.hpp

```diff
@@ -12,32 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef DWB_CONTROLLER__DWB_CONTROLLER_HPP_
-#define DWB_CONTROLLER__DWB_CONTROLLER_HPP_
+#ifndef NAV2_CONTROLLER__NAV2_CONTROLLER_HPP_
+#define NAV2_CONTROLLER__NAV2_CONTROLLER_HPP_
 
 #include <memory>
 #include <string>
 #include <thread>
 
-#include "dwb_core/common_types.hpp"
-#include "dwb_core/dwb_local_planner.hpp"
+#include "nav2_core/local_planner_base.hpp"
+#include "nav2_costmap_2d/costmap_2d_ros.hpp"
+#include "tf2_ros/transform_listener.h"
 #include "nav_2d_msgs/msg/pose2_d_stamped.hpp"
+#include "nav2_msgs/action/follow_path.hpp"
+
 #include "nav_2d_utils/odom_subscriber.hpp"
 #include "nav2_util/lifecycle_node.hpp"
-#include "nav2_msgs/action/follow_path.hpp"
 #include "nav2_util/simple_action_server.hpp"
 #include "nav2_util/robot_utils.hpp"
 
-namespace dwb_controller
+#include "pluginlib/class_loader.hpp"
+#include "pluginlib/class_list_macros.hpp"
+
+namespace nav2_controller
 {
 
 class ProgressChecker;
 
-class DwbController : public nav2_util::LifecycleNode
+class nav2_controller : public nav2_util::LifecycleNode
```

Comment:
`Nav2Controller` since this is a class not a namespace. Also the package is named nav2_controller, the controller server doesnt have to, necessarily. `ros2 run nav2_controller controller_server` has a ring to it, but I"m not the best namer

Reply:
Changed class name to Nav2Controller.